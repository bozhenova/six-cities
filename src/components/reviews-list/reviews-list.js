import React from 'react';
import PropTypes from 'prop-types';
import Review from '../review/review';
const shortid = require('shortid');

const ReviewsList = props => {
  const reviews = offers.map(offer => {
    return (
      <li className='reviews__item' key={shortid.generate()}>
        <Review />
      </li>
    );
  });

  return <ul class='reviews__list'>{reviews}</ul>;
};

export default ReviewsList;
