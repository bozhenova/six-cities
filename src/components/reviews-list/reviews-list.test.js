import React from 'react';
import { ReviewsList } from './reviews-list';
import renderer from 'react-test-renderer';

describe('Reviews List', () => {
  it('should render a list of reviews', () => {
    const tree = renderer.create(<ReviewsList reviews={[]} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
