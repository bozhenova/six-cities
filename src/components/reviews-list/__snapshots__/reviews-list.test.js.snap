// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews List should render a list of reviews 1`] = `
<ul
  className="reviews__list"
/>
`;
